parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -91,6 +91,34 @@ ensureAuditD() {
   fi
 }
 
+{{- if ShouldConfigTransparentHugePage}}
+configureTransparentHugePage() {
+    THP_CONFIG_PATH="/etc/sysfs.conf"
+    THP_ENABLED={{GetTransparentHugePageEnabled}}
+    if [[ "${THP_ENABLED}" != "" ]]; then
+        echo "${THP_ENABLED}" > /sys/kernel/mm/transparent_hugepage/enabled
+        echo "kernel/mm/transparent_hugepage/enabled=${THP_ENABLED}" >> ${THP_CONFIG_PATH}
```

Comment:
just curious - do you know why we need the same setting in two places?

Reply:
The second sysfs setting is to make it persistent after reboot